src/main/java/seedu/address/logic/commands/commandresults/DisplayableCommandResult.java

```diff
@@ -86,14 +92,16 @@ public boolean equals(Object other) {
             return feedbackToUser.equals(otherCommandResult.feedbackToUser)
                     && showHelp == otherCommandResult.showHelp
                     && exit == otherCommandResult.exit
-                    && personToDisplay.get() == otherCommandResult.personToDisplay.get();
+                    && personToDisplay.get() == otherCommandResult.personToDisplay.get()
```

Comment:
I forgot to change this the other time but perhaps this comparison should be made using `.equals` instead of `==` since they are Objects that are being compared?